README.md

```diff
@@ -1,30 +1,44 @@
 # UnJS Docs
 
-Minimal Documentation Theme and CLI for shared usage across unjs projects.
+Minimal Documentation Theme and CLI for shared usage across UnJS projects.
 
 Made with [Nuxt](https://nuxt.com/), [Nuxt Content](https://content.nuxt.com) and [Nuxt UI Pro](https://ui.nuxt.com/pro).
 
+This is fully SEO friendly thanks to [Nuxt SEO](https://nuxtseo.com).
+
 ## Usage
 
 > [!NOTE]
 > This project is under development and usage might change.
 
 1. Install `unjs-docs` as a dev dependency
-2. Create `app.config.ts`:
+2. Create `docs.config.ts` and set the name, description, github and themeColor.
 
 ```ts
-export default {
-  docs: {
-    name: '...',
-    description: '...',
-    github: '.../...',
-  },
-}
+import { defineDocsConfig } from 'unjs-docs/config'
+
+export default defineDocsConfig({
+  name: 'Docs Theme',
+  description: 'Default documentation for UnJS package.',
+  github: 'unjs/docs',
+  themeColor: '#f98007',
+})
 ```
 
-3. Create `content/index.md` or `content/index.yml`
+> [!NOTE]
+> Color can be a HEX or RGB value.
+
+3. Create `content/index.yml` for the homepage and `content/**.md` for the docs.
+
+> [!IMPORTANT]
+> You **must** set a description in the front-matter of each markdown file.
```

Comment:
It shouldn't be a requirement. Our tools #1 goal is to support writing docs without any specific tooling requirement like a normal markdown at any costs

Reply:
Mmmmh, ok, I will try to deal with that.

Reply:
My main concern is that a subtitle is not a paragraphe like any other.

For example:
![image](https://github.com/unjs/docs/assets/45267552/e2f82793-5209-4594-897d-6db5a5112838)

Description starts with an action and this can't be the first sentence of the document. Now, imagine the first sentence is the subtitle, it's strange for the reader.

Reply:
This is weird. And if we think that the first p disappear thanks to our tool we are not in the case of a "not specific tooling".

![image](https://github.com/unjs/docs/assets/45267552/b606d997-3ca8-4878-a910-23fa517e24be)

The description could be remove from the final page, this is not a big deal for the user since it's here to be a subtitle and the open graph description. So having it in the frontmatter makes not it tooling agnostic but having a tool that does not support front-matter will not be a big deal.

Reply:
What bothers me is that there should not be a MUST. A normal markdown should work without problems to write docs and additions like SEO description should be either automated (if it is smart enough) or opt-in (as later enhancement to the docs). I don't want to need this for making every documentation page. it is a pain to do and slows down to write more docs. SEO is important too, writing docs without hassle is most concern.

Reply:
Ok, I understand the point now. will update #22 and make it optional!